packages/suite/src/middlewares/wallet/fiatRatesMiddleware.ts

```diff
@@ -51,23 +51,19 @@ const fiatRatesMiddleware = (api: MiddlewareAPI<Dispatch, AppState>) => (next: D
         case ACCOUNT.CREATE: {
             // fetch current rates for account's tokens
             const account = action.payload;
-            if (account.tokens) {
-                account.tokens.forEach((t, i) => {
-                    if (t.symbol) {
-                        const s = t.symbol;
-                        // wait 500ms before firing next fetch
-                        setTimeout(() => {
-                            api.dispatch(
-                                fiatRatesActions.updateCurrentRates({
-                                    symbol: s,
-                                    mainNetworkSymbol: account.symbol,
-                                    tokenAddress: t.address,
-                                }),
-                            );
-                        }, i * 500);
-                    }
-                });
-            }
+            account.tokens?.forEach(t => {
```

Comment:
nit:
```suggestion
            account.tokens?.forEach(token => {
```
I would rather see here `token` instead of just `t`

Reply:
you will get use to it! 😁 Or you can refactor it across the suite later (also `a` for an account) 😛

Reply:
I agree with @tomasklim that descriptive names are usually better for readability than abbreviations. I.e. I lean bit more towards Java rather than C.

However there are many cases where shorter identifiers make a difference, especially in these functional-style collection-processing pipelines like `forEach`, `filter`, `map`...

In JS the developer have to name the variable themselves every time. I like how this is done in Kotlin where the scoped variable is implicitly named `it` in all functions that accept a lambda. For example the code above would be written as this in Kotlin
```kotlin
account.tokens?.foreach {
    api.dispatch(it.symbol)
}
```

Note that we haven't defined the `it` variable anywhere. It was automatically provided to the scope by the `forEach` function. The variable can still be renamed. eg. like this
```kotlin
account.tokens?.foreach { token ->
    api.dispatch(token.symbol)
}
```

which is handy especially when there're multiple nested scopes.

The nice thing about implicit `it` is that with basic knowledge of the language one can immediately know what does it refer to -- the subject of the closest lambda scope. In javascript the reader must remember where each of these abbreviated variables were defined and keep it in mind.

In my wild imagination we'd have a convention that we name all left-most lambda arguments `it` except for cases where a longer name makes more sense. This could prevent review threads as this one and would make the code a bit more readable. It has one catch tough: the convention must be upheld across the codebase, which in itself is quite hard without a help of some custom linter rules.

---

I digress. I vote for `token` here. .)

Reply:
👀 , kay... let's have `token` here since it seems it is really important for you 😁